src/App/Fossa/Config/Analyze.hs

```diff
@@ -447,6 +451,17 @@ mergeStandardOpts maybeConfig envvars cliOpts@AnalyzeCliOpts{..} = do
     <*> pure grepOptions
     <*> pure customFossaDepsFile
 
+collectMavenScopeFilters ::
+  ( Has Diagnostics sig m
+  ) =>
+  Maybe ConfigFile ->
+  m MavenScopeFilters
+collectMavenScopeFilters maybeConfig = do
+  let cfgMavenScopeFilters = maybe mempty collectConfigMavenScopeFilters maybeConfig
```

Comment:
[Prefer, Optional] you can simplify by doing: `pure $ maybe mempty collectConfigMavenScopeFilters maybeConfig`